pkg/query/api/v1.go

```diff
@@ -447,27 +496,28 @@ func (api *API) series(r *http.Request) (interface{}, []error, *apiError) {
 		matcherSets = append(matcherSets, matchers)
 	}
 
+	enableDedup, aerr := api.parseEnableDedupParam(r)
+	if aerr != nil {
+		return nil, nil, aerr
+	}
+
+	enablePartialResponse, aerr := api.parsePartialResponseParam(r)
+	if aerr != nil {
+		return nil, nil, aerr
+	}
+
 	var (
-		warnmtx             sync.Mutex
-		warnings            []error
-		enableDeduplication = true
+		warnmtx  sync.Mutex
+		warnings []error
 	)
-	partialErrReporter := func(err error) {
+	warningReporter := func(err error) {
```

Comment:
Not sure, how this would be helpful. Warning can be for any reason, for example if no store matches, that might be valid case for wrong query, but warning helps to understand the problem.

I would consider adding metric when we return partial response though, that might be helpful

Reply:
Although wonder if grpc client requests fails is not incremented then.. will double check